demos/sankey.html

```diff
@@ -3,20 +3,42 @@
   <head>
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title>States Mermaid Quick Test Page</title>
+    <title>Sankey Mermaid Quick Test Page</title>
     <link rel="icon" type="image/png" href="data:image/png;base64,iVBORw0KGgo=" />
-    <style>
-      div.mermaid {
-        /* font-family: 'trebuchet ms', verdana, arial; */
-        font-family: 'Courier New', Courier, monospace !important;
-      }
-    </style>
   </head>
 
   <body>
     <h1>Sankey diagram demos</h1>
+    <h2>FY20-21 Performance</h2>
+    <pre class="mermaid">
+      ---
+      config:
+        sankey:
+          showValues: true
+          prefix: $
+          suffix: B
+          width: 800
+          nodeAlignment: left
```

Comment:
@aloisklink I have a query before we release this. Do we need the config key?
The only other keys in frontmatter are title and displayMode, with the addition of config, the chance of us adding a top level frontmatter key is slim. 

So, should we remove the `config` key, and directly write the config at the root?

Reply:
Discussion moved to https://github.com/mermaid-js/mermaid/issues/4764